omnichain/staking/contracts/Staking.sol

```diff
@@ -9,15 +9,17 @@ import "@zetachain/toolkit/contracts/BytesHelperLib.sol";
 contract Staking is ERC20, zContract {
     error SenderNotSystemContract();
     error WrongChain();
-    error NotAuthorizedToClaim();
+    error UnknownAction();
 
     SystemContract public immutable systemContract;
     uint256 public immutable chainID;
```

Comment:
I think immutable could be written in full uppercase.

Reply:
that's not the solidity standard and is not the one with use. is more important to be consistent with all the contracts.